packages/nimble-components/src/rich-text/editor/models/create-tiptap-editor.ts

```diff
@@ -110,7 +110,7 @@ export function createTiptapEditor(
  * "parseHTML": https://tiptap.dev/guide/custom-extensions#parse-html
  * "renderHTML": https://tiptap.dev/guide/custom-extensions/#render-html
  */
-function createCustomLinkExtension(): Mark<LinkOptions> {
+function createCustomLinkExtension(): Mark {
```

Comment:
With this and the other change in the file, we could have used an `as` cast, but if none of the callers need a more constrained type, it doesn't seem necessary.